sphinxext/opengraph/__init__.py

```diff
@@ -72,7 +72,7 @@ def get_tags(
         parse_result = urlparse(config["html_baseurl"])
 
         if config["html_baseurl"] is None:
-            raise EnvironmentError("ReadTheDocs did not provide a valid canonical URL!")
```

Comment:
Why was this changed?

Reply:
I ran https://github.com/asottile/pyupgrade/ to upgrade to modern syntax:

According to the [Python docs](https://docs.python.org/3/library/exceptions.html?highlight=environmenterror#EnvironmentError), `EnvironmentError` is kept for compatibility with previous versions; starting from Python 3.3, it's an alias of `OSError`. So we can use `OSError` instead.